modules/alibaba-ai-converting-image-to-qcow2.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_alibaba/installing-alibaba-assisted-installer.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="alibaba-ai-converting-image-to-qcow2_{context}"]
+= Converting the Discovery image to QCOW2 format
+
+Convert the generated ISO to `QCOW2` format before importing it into {alibaba}.
+
+.Prerequisites
+
+* You have created a cluster and downloaded the Discovery image in {ai-full}.
+
+.Procedure
+
+. Open the command line interface. 
```

Comment:
First step is usually telling the user which cluster they need to log in to. Otherwise, they can't do anything with the 'oc' tool.